catalog/core/catalog-core-directorymonitor/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -22,13 +22,18 @@
             type="String" default=""/>
 
         <AD description="Optional: Copy the ingested files into a backup directory under the monitored directory named /.ingested  -  NOTE: this will double the amount of disk space required for ingested files in this monitored directory if its Processing Directive includes storing the file in the DDF Content Repository."
-            name="Copy Files to Backup Directory" id="copyIngestedFiles" required="false"
-            type="Boolean" default="false"/>
+            name="Processing Mechanism" id="processingMechanism" required="false"
```

Comment:
Can we update the description to go with these new options? There has been some confusion on whether copying the ingest files actually still moves them. Basically move will move rather than copy the files to that backup directory, correct? The description is confusing because admins expect the files to still be there since it says it `copies` the ingested files.

@kcwire looks like some of the confusion on the directory monitor configuration is already cleaned-up in the drop down.